recipes-ni/cgroups-v2/files/cgroupv2-common.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+# Common cgroup v2 paths
+CGROUP2_ROOT="/sys/fs/cgroup"
+NI_GROUP="${CGROUP2_ROOT}/ni"
+SUBGROUPS="irq scanengine system time-critical timed-structures"
+
+setup_mount_and_chown() {
+    if ! mountpoint -q "${CGROUP2_ROOT}"; then
+        mount -t cgroup2 none "${CGROUP2_ROOT}"
+    fi
+
+    cd "${CGROUP2_ROOT}"
+
+    echo "+cpuset" > cgroup.subtree_control
+
+    mkdir -p "${NI_GROUP}"
+    chown lvuser:ni "${NI_GROUP}"
+
+    cd "${NI_GROUP}"
+
+    echo "+cpuset" > cgroup.subtree_control
+
+    for group in $SUBGROUPS; do
+        mkdir -p "$group"
+        chown lvuser:ni "$group"
```

Comment:
Can you double check that the files under these folders also end up being owned by `lvuser:ni` or at least the ones that LabVIEW needs access to.

Otherwise we may need a recursive chown at the end of this function instead, i.e. `chown -R lvuser:ni "${NI_GROUP}"`

Reply:
Only the directories under `/sys/fs/cgroup/ni` had the owner set to `lvuser:ni` and not the files. I have changed this to `chown -R` now, and all the files now belong to `lvuser:ni` as well.